UKHomeOffice/sgmr-service
# SPCR-488 Fix id's on skipper page to be unique

### AC / Description of changes

Some fields had their div and their input ID’s set to `<fieldname>`
when we want them to be div `<fieldname>` and input `<fieldname>Input`

Updated these, and updated the corresponding htmlFor on the label element

### To Test

- go to skipper page on voyage
- run axe dev tools
- > there should be no issues related to IDs or field labels


### Notes

---

**Remember to merge to feature branches not `master`.**
